src/material.h

```diff
@@ -136,7 +136,9 @@ class Material: public Resource {
   /// not result in an updated material composition.  Does nothing if the
   /// simulation decay mode is set to "never" or none of the nuclides' decay
   /// constants are significant with respect to the time delta.
-  void Decay(int curr_time);
+  /// @param curr_time current time to use for the decay calculation 
+  ///        (default: -1 forces the decay to the context's current time)
```

Comment:
@gonuke this is from #1687

Reply:
Currently, a user/developer can request a decay to any time of interest.  If they give no time, it will be set to -1 and interpreted as the current time.